README.md

```diff
@@ -1109,8 +1111,8 @@ The `.widgetrc` file is a configuration file that saves your local widget settin
 
 | Command | Description |
 | --- | --- |
-| `npm start` | Build the widget, start the server, and open a browser window with the widget loaded |
-| `npm run build:dev` | Build an unminified version of the widget |
-| `npm run build:release` | Build a minified, uglified version of the widget |
-| `npm test` | Run unit tests |
-| `npm run lint` | Run jshint and scss linting tests |
+| `yarn start` | Build the widget, start the server, and open a browser window with the widget loaded |
+| `yarn run build:dev` | Build an unminified version of the widget |
```

Comment:
In this case, yarn seems to not need the "run" part (should verify this), but I think we can replace this with just `yarn build:dev`, `yarn build:release`, `yarn lint`